src/vip-framework/index.ts

```diff
@@ -66,6 +70,7 @@ const executeCommand = async (timelock: SignerWithAddress, proposal: Proposal, c
     value: proposal.values[commandIdx],
     data: encodeMethodCall(proposal.signatures[commandIdx], proposal.params[commandIdx]),
     gasLimit: 8000000,
+    // maxFeePerGas: 2000000000, // Needed for Zksync
```

Comment:
We could add this value to the networkConfig.ts file, only setting a value for the zkSync networks. wdyt?

Reply:
[6b38c2f](https://github.com/VenusProtocol/vips/pull/326/commits/6b38c2f037f2a89746d07ced8b455e6a7877d73d)